test/plugin/test_output_as_buffered.rb

```diff
@@ -262,9 +288,12 @@ def waiting(seconds)
       end
     end
 
-    test 'plugin using custom format cannot iterate chunk in #write' do
+    data('formatted_to_msgpack_binary?' => :custom,
+         'formatted_to_msgpack_binary' => :old_custom)
+    test 'plugin using custom format cannot iterate chunk in #write' do |out_type|
```

Comment:
This testcase doesn't verify this change because `@format_type_is_msgpack.call` always returns `false` regardless of overriding `#formatted_to_msgpack_binary?`.